### PR TITLE
cleanup: Remove teleportd download in VHD

### PR DESCRIPTION
**Pull request by ganeshkumarashok**
**What type of PR is this?**
Teleportd is not supported anymore. 

**What this PR does / why we need it**:
Removing it to reduce VHD space and cleanup. This is also towards unblocking the weekly VHD build. 

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:

```
none
```

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -353,12 +353,10 @@ KUBERNETES_VERSION=$CRICTL_VERSIONS installCrictl || exit $ERR_CRICTL_DOWNLOAD_T
 ctr namespace create k8s.io
 cliTool="ctr"
 
-# also pre-download Teleportd plugin for containerd
-downloadTeleportdPlugin ${TELEPORTD_PLUGIN_DOWNLOAD_URL} "0.8.0"
 
 INSTALLED_RUNC_VERSION=$(runc --version | head -n1 | sed 's/runc version //')
 echo "  - runc version ${INSTALLED_RUNC_VERSION}" >> ${VHD_LOGS_FILEPATH}
-capture_benchmark "artifact_streaming_and_download_teleportd"
+capture_benchmark "artifact_streaming_and_download"
```
Comment (anujmaheshwari1) [inline vhdbuilder/packer/install-dependencies.sh:359]: nit: whats the download at the end for? rename?

**Review by anujmaheshwari1 — APPROVED** [review]

File: vhdbuilder/packer/install-dependencies.sh (hunk 1 above)
Comment (ganeshkumarashok) [reply vhdbuilder/packer/install-dependencies.sh:359]: this is for naming the steps during VHD build time benchmarking

Comment (ganeshkumarashok) [reply vhdbuilder/packer/install-dependencies.sh:359]: can rename

Comment (anujmaheshwari1) [reply vhdbuilder/packer/install-dependencies.sh]: i got that, i was saying "artifact_streaming_and_download" seems off, made sense when it was "download_teleportd".

am i misunderstanding?

Comment (ganeshkumarashok) [reply vhdbuilder/packer/install-dependencies.sh]: right, renamed it to: `artifact_streaming_download`

**Review by UtheMan — APPROVED** [review]

**Review by cameronmeissner — APPROVED** [review]

**Pull request merged**